src/object/trigger_manager.c

```diff
@@ -6866,6 +6900,12 @@ tr_rename_trigger (DB_OBJECT * trigger_object, const char *name, bool call_from_
 		  (void) trigger_table_rename (trigger_object, oldname);
 		  oldname = NULL;
 		}
+
+	      if (error == NO_ERROR)
+		{
+		  error = locator_flush_instance (trigger_object);
+		}
+
```

Comment:
It seems to have to be placed before line 6890 if (error != NO_ERROR).

Reply:
I'm changing the tr_rename_trigger function a bit.
I will consider it together.

Reply:
Please review the changed tr_rename_trigger() function again.

Reply:
(void) trigger_table_rename (trigger_object, oldname); should be called even if locator_flush_instance() failed.